src/super_gradients/module_interfaces/export_params.py

```diff
@@ -0,0 +1,74 @@
+import dataclasses
+from typing import Optional, Tuple
+
+from super_gradients.conversion import ExportTargetBackend, DetectionOutputFormatMode
+
+
+@dataclasses.dataclass
+class ExportParams:
+    """
+    Parameters for exporting a model to ONNX format in PTQ/QAT methods of Trainer.
```

Comment:
No it will not. Indeed, there is a major overlap in parameters in `ExportParams` and in `model.export` arguments. However it's not 100% match.
1) For instance, when we are doing PTQ/QAT we by default working in INT8 so it has no sense to have this parameter in `ExportParams` as it's hard-coded and fixed by design. Having this option in `ExportParams` makes not sense.
1) An `export()` for segmentation models differs from `export` for OD and there are less arguments to control the postprocessing step. So there is no sense in having same dataclass where some fields would not be used.
2) In `ExportParams` we have to account of all possible model types, so this dataclass becomes **almighty** container to contain all possible options. This is not the cleanest design but frankly I could not find a better options to offer some schema and support loading export params from YAML at the same time.

Reply:
ok.
We might want to have Export params in the future, and have this class inherit from it.
So maybe name it QuantizedExportParams (we will inherit from ExportParams in the future)

Reply:
As agreed, I've moved ExportParams to `super_gradients.conversion` namespace and the rest will happen some day :)